configs/pruning/mmcls/dcff/fix_subnet.json

```diff
@@ -14,6 +14,102 @@
                     "min_value":1,
                     "min_ratio":0.9
                 },
+                "channels":{
```

Comment:
Could the "channels" field be hidden when exporting and loading subnet?

Reply:
Subject to the way channel_unit is initialized, the pruned model can only be constructed at this stage if the channel is included at export and load time. At least for pruning algorithms.